templates/summit/schedule.php

```diff
@@ -75,33 +78,33 @@
 
 <div class="keynote-speakers">
 
-  <div class="speaker">
-    <a href="https://tantek.com/"><img src="/images/tantek.jpg"></a>
-    <p class="name">Tantek<br> Çelik</p>
+  <div class="speaker h-card">
```

Comment:
What's the purpose of adding an `h-card` here? is there a consuming use case? I would rather not have random microformats on the page if there isn't a plan to consume it from something, since it may confuse things like post type discovery.

Reply:
tl;dr: Dropped speaker h-cards pending further discussion. The consuming use-case is to be able to more quickly/easily add keynote speakers to your address book, however that's orthogonal from the rest of the fixes, so dropping h-card pending further discussion/consensus.